android/app/src/main/java/com/cBreez/client/BreezFCMService.kt

```diff
@@ -0,0 +1,73 @@
+package com.cBreez.client
+
+import android.util.Log
+import androidx.work.BackoffPolicy
+import androidx.work.Constraints
+import androidx.work.ExistingWorkPolicy
+import androidx.work.NetworkType
+import androidx.work.OneTimeWorkRequest
+import androidx.work.OneTimeWorkRequestBuilder
+import androidx.work.OutOfQuotaPolicy
+import androidx.work.WorkManager
+import androidx.work.WorkRequest.Companion.DEFAULT_BACKOFF_DELAY_MILLIS
+import androidx.work.workDataOf
+import com.google.firebase.messaging.FirebaseMessagingService
+import com.google.firebase.messaging.RemoteMessage
+import java.util.concurrent.TimeUnit
+
+
+class BreezFCMService : FirebaseMessagingService() {
+    val TAG = "BreezFCMService"
+
+    override fun onMessageReceived(remoteMessage: RemoteMessage) {
+        Log.d(TAG, "From: ${remoteMessage.from}")
+
+        // Check if message contains a data payload.
```

Comment:
Is this logic in Breez mobile needed to still be able to handle notifications in flutter? https://github.com/breez/breezmobile/blob/master/android/app/src/main/java/com/breez/client/BreezFirebaseMessagingService.java#L48